api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantLifecycleIntegrationTests.java

```diff
@@ -0,0 +1,230 @@
+            throws NotificationClientException, JsonProcessingException {
+        for (int i = 0; i < numCycles; i++) {
+            resetNotificationClient();
+            repositoryMediator.fastForwardDatabaseEntities(CYCLE_DURATION);
+            wiremockManager.stubSuccessfulEligibilityResponse(NO_CHILDREN);
+            invokeAllSchedulers();
+            assertRegularPaymentCyclePaidCorrectly(claimId, NO_CHILDREN);
+            verifyNoMoreInteractions(notificationClient);
+        }
+        return expectedDeliveryDate.minusDays((long) CYCLE_DURATION * numCycles);
+    }
+
+    private List<LocalDate> progressThroughCycleWithUpcomingBirthday(UUID claimId, List<LocalDate> childrenDob, EmailType emailType)
+            throws NotificationClientException, JsonProcessingException {
+        resetNotificationClient();
+        childrenDob = fastForwardOneCycle(childrenDob);
+        PaymentCycle paymentCycle = assertRegularPaymentCyclePaidCorrectly(claimId, childrenDob);
+        assertEmailSent(emailType, paymentCycle);
+        verifyNoMoreInteractions(notificationClient);
+        return childrenDob;
+    }
+
+    private List<LocalDate> progressThroughRegularPaymentCycles(UUID claimId, List<LocalDate> childrenDob, int numCycles) throws NotificationClientException,
+            JsonProcessingException {
+        for (int i = 0; i < numCycles; i++) {
+            resetNotificationClient();
+            childrenDob = fastForwardOneCycle(childrenDob);
+            assertRegularPaymentCyclePaidCorrectly(claimId, childrenDob);
+            verifyNoMoreInteractions(notificationClient);
+        }
+        return childrenDob;
+    }
+
+    private UUID applyForHealthyStartAsPregnantWomanWithNoChildren(LocalDate expectedDeliveryDate) throws JsonProcessingException, NotificationClientException {
+        String cardAccountId = UUID.randomUUID().toString();
+        wiremockManager.stubSuccessfulEligibilityResponse(NO_CHILDREN);
+        wiremockManager.stubSuccessfulNewCardResponse(cardAccountId);
+        wiremockManager.stubSuccessfulCardBalanceResponse(cardAccountId, 0);
+        wiremockManager.stubSuccessfulDepositResponse(cardAccountId);
+        stubNotificationEmailResponse();
+
+        ClaimDTO claimDTO = aClaimDTOWithClaimant(aClaimantDTOWithExpectedDeliveryDate(expectedDeliveryDate));
+        restTemplate.exchange(buildClaimRequestEntity(claimDTO), ClaimResultDTO.class);
+        invokeAllSchedulers();
+
+        ClaimantDTO claimant = claimDTO.getClaimant();
+        Claim claim = repositoryMediator.getClaimForNino(claimant.getNino());
+        return claim.getId();
+    }
+
+    private List<LocalDate> fastForwardOneCycle(List<LocalDate> initialDates) throws JsonProcessingException {
+        repositoryMediator.fastForwardDatabaseEntities(CYCLE_DURATION);
+        List<LocalDate> newDates = initialDates.stream().map(localDate -> localDate.minusDays(CYCLE_DURATION)).collect(Collectors.toList());
```

Comment:
Can we rename `initialDates` to `initialChildrenDobs`, I was initially unsure what dates we were talking about.

Also, why are we changing the dates of birth, surely they need to stay constant once they have been created? I might be getting confused again here...